src/generator/AutoRest.Go/CodeNamerGo.cs

```diff
@@ -37,43 +37,43 @@ public new static CodeNamerGo Instance
         // CommonInitialisms are those "words" within a name that Golint expects to be uppercase.
         // See https://github.com/golang/lint/blob/master/lint.go for detail.
         private string[] CommonInitialisms => new string[] {
```

Comment:
Probably should change this to a HashSet<string> that uses StringComparer.OrdinalIgnoreCase for its IEqualityComparer.